pkg/runtime/application_config_provider.go

```diff
@@ -38,7 +39,7 @@ func (p *ApplicationConfigurationProvider) GetDbConfig() dbconfig.DbConfig {
 			logger.Fatalf(context.Background(), "failed to read database password from path [%s] with err: %v",
 				dbConfigSection.PasswordPath, err)
 		}
-		password = string(passwordVal)
+		password = strings.TrimSpace(string(passwordVal))
```

Comment:
Can you link to https://www.postgresql.org/docs/current/libpq-connect.html and mention that passwords can contain special characters as long as they are percent-encoded?

Reply:
Sure, do you want me to add it in file directly as a comment or somewhere else?

Reply:
I added it as a comment.